scripts/validate-js.py

```diff
@@ -14,17 +14,24 @@
     print 'Usage: ' + sys.argv[0] + ' file.json+'
     exit(0)
 
+def exclude_serverless_function(value):
+    if value.startswith('function_') or value.startswith('functions'):
+        return False
+    else:
+        return True
+
 def fmap(F, data):
     if isinstance(data, dict):
         for key, value in data.iteritems():
             fmap(F, (key,value))
     elif isinstance(data, tuple):
         (key, value) = data
         if isinstance(value, basestring):
-            if value.startswith('function'):
+            if value.startswith('function') and exclude_serverless_function(value):
                 F(data)
         elif isinstance(value, list) and len(value) > 0 \
-             and isinstance(value[0], basestring) and value[0].startswith('function'):
+             and isinstance(value[0], basestring) and value[0].startswith('function') \
+             and exclude_serverless_function(value[0]):
```

Comment:
A general naming convention for functions that return Booleans - `is_` and `should_` prefixes.

`is_valid`: returns if arguments are valid, for instance.
`should_exclude_serverless(value[0])`: returns whether to exclude the argument